a10_octavia/controller/worker/tasks/decorators.py

```diff
@@ -30,6 +43,10 @@ def wrapper(self, *args, **kwargs):
             self.axapi_client = acos_client.Client(vthunder.ip_address, api_ver,
                                                    vthunder.username, vthunder.password,
                                                    timeout=30)
+            if vthunder.topology == constants.ROLE_STANDALONE:
```

Comment:
If it has a partition in the first place on the config then we know it's a racked device. We don't have support for this in vThunders.

```python
if vthunder.partition != a10constants.SHARED_PARTITION
```

Should suffice as it's not an option for vThunders

Reply:
Added it assuming we are doing it for non hardware devices too. Will remove it